privly.safariextension/scripts/background_scripts/handle_context_menu.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * fileOverview This file handles the context menu item, "Privly New Message"
```

Comment:
Changes made in https://github.com/sammyshj/privly-safari/commit/ab5bd109dbdb4ac09eb60ac35491b86d9c9a8e23